themes-default/slim/src/store/modules/shows.js

```diff
@@ -0,0 +1,50 @@
+import { ADD_SHOW } from '../mutation-types';
+
+const state = {
+    shows: []
```

Comment:
I believe it will result in a store state of:
```js
store: {
	state: {
		shows: { // module
			shows: [] // module state
		}
	}
}
```

Reply:
This will still work so maybe it's not too bad.

```js
...mapState('shows', [
   'shows'
]),
```

Reply:
The working solution is:
```js
    computed: {
        ...mapState({ shows: state => state.shows.shows }),
        localComputed() { ... }
    }
```